src/test/java/feiner/projectile/ProjectileTest.java

```diff
@@ -31,13 +27,11 @@ public void testGetY() {
         // then
         assertEquals(-7.90, actual, 0.01);
     }
-
     @Test
     public void testGetApexTime() {
         // given
         Projectile projectile = new Projectile(31, 20);
 
-
```

Comment:
Here I just wanted you to get rid of the extra line.
Not all the new lines in your classes.

Reply:
I fixed it so that I only got rid of one extra line.